code/api/package-lock.json

```diff
@@ -461,6 +461,32 @@
         "@babel/helper-plugin-utils": "^7.8.0"
       }
     },
+    "@babel/plugin-syntax-bigint": {
```

Comment:
I'm not sure if this is going to be a an issue at all, but will we be running into an issue if this file is still updating on each PR?
Mostly did ya'll pull/rebase the most recent code on mainbefore this, because if so this should just be additions for future code and not an issue?

Reply:
I'm wondering if it would be more efficient to use `npm install` after each pull from `main` instead of pushing up so many lines every time we merge. We should discuss the pros and cons of both.

Reply:
@nathanielmillard  & @kathrynljackson  yeah you both raise good points. Did we pull/rebase before this? No. But it wouldn't be a problem to do it. I'm curious what `npm install` would do for us. We should definitely talk